controllers/contentlibrary/v1alpha2/clustercontentlibraryitem/clustercontentlibraryitem_controller.go

```diff
@@ -258,6 +260,33 @@ func (r *Reconciler) setUpCVMIFromCCLItem(ctx *context.ClusterContentLibraryItem
 	cvmi.Status.Name = cclItem.Status.Name
 	cvmi.Status.ProviderItemID = string(cclItem.Spec.UUID)
 
+	return addContentLibraryRefToAnnotation(ctx)
+}
+
+// addContentLibraryRefToAnnotation adds the conversion annotation with the content
+// library ref value populated.
+func addContentLibraryRefToAnnotation(ctx *context.ClusterContentLibraryItemContextA2) error {
```

Comment:
We could probably dedupe the two nearly identical implementations of this into  `controllers/contentlibrary/v1alpha2/utils` packages by using `client.Object` but that isn't a huge deal.

Reply:
I will address this in a quick follow up one